test/upload-file-test.js

```diff
@@ -187,6 +188,57 @@ suite.addBatch({
                                     assert.ifError(err);
                                 }
                             },
+                            "and we get the file from web interface without login": {
+                                topic: function(doc, feed, pair, cl) {
+                                    var cred = makeCred(cl, pair),
+                                        browser = new Browser(),
+                                        callback = this.callback,
+                                        url = doc.fullImage.url;
+
+                                    browser.visit(url, function(err) {
+                                        callback(err, browser);
+                                    });
+                                },
+                                "it fails correctly": function(err, br) {
+                                    br.assert.status(403);
+                                }
+                            },
+                            "and we login for try get file": {
+                                topic: function(doc, feed, pair, cl) {
+                                    var browser = new Browser(),
+                                        callback = this.callback,
+                                        user = pair.user;
+
+                                    browser.visit("http://localhost:4815/main/login")
+                                        .then(function() {
+                                            browser.fill("#nickname", user.nickname)
+                                                .fill("#password", "stormtroopers_hittin_the_ground")
+                                                .pressButton("div#loginpage form button[type=\"submit\"]", function() {
+                                                    callback(null, browser);
+                                                }, callback);
+                                        }, callback);
+                                },
+                                "it works": function(err, br) {
+                                    assert.ifError(err);
+                                    br.assert.success();
+                                },
+                                "and we get the file from web interface with login": {
```

Comment:
s/with login/while logged in/

Reply:
And add a "the" before "web interface" too.

Reply:
Done.